wire-runtime-swift/src/test/swift/sample/Dinosaur.swift

```diff
@@ -15,6 +15,7 @@ public struct Dinosaur {
     public var picture_urls: [String] = []
     public var length_meters: Double?
     public var mass_kilograms: Double?
+    @Defaulted(defaultValue: Period.CRETACEOUS)
```

Comment:
this is weird why isnt the rest of the fields `length_meters` defaulted...? looking.

Reply:
figured it out has to do with optionality on the typeName...fixing